src/modules/UI/components/ControlPanel/Component/Main.js

```diff
@@ -164,7 +164,7 @@ const ScanButton = () => {
 
 const SweepPrivateKeyButton = () => {
   return (
-    <Button onPress={Actions.scan}>
+    <Button onPress={() => Actions.scan('sweepPrivateKey')}>
```

Comment:
Do not pass inline functions as a prop. This will force a rerender. Instead create another function in the class and pass that in